### ISSUE TITLE
FAQ for using Rmd

### ISSUE THREAD

**Issue by aghaynes** [CTU-Bern/CTUtemplate#35]
- [ ] add a bibliography
- [ ] force placement of a figure
- [ ] make a page landscape? (already in one of the templates)
- [ ] saving output to a different folder? (rmarkdown::render or onEnd (not sure of the function name))
- [ ] add other latex packages
- [ ] ...

**Comment by aghaynes** [comment]
Saving files to another location... 
https://stackoverflow.com/questions/28894515/rmarkdown-directing-output-file-into-a-directory

**Comment by aghaynes** [comment]
@ArnaudKunzi I was thinking of changing the template such that :point_up: is in there already... what do you think?

**Comment by ArnaudKunzi** [comment]
Agreed, it will be more concise to document this as a YAML option with the added benefit that it is stated at the top of the file.

**Comment by aghaynes** [comment]
maybe this is not so relevant anymore if we're switching to quarto and have [the code library page](https://github.com/CTU-Bern/CTU-code-library/blob/main/misc/quarto.md)?
